cvs -z3 diff scrlwing.cpp (in directory E:\Dev\wxWindows\src\generic\)
Index: scrlwing.cpp
===================================================================
RCS file: /home/wxcvs/wxWindows/src/generic/scrlwing.cpp,v
retrieving revision 1.8
diff -r1.8 scrlwing.cpp
275a276,277
>     if ( m_win && m_win != m_targetWindow)
>         m_win->PopEventHandler(TRUE /* do delete it */);
360a363,371
> 
>     // FIXME: There is a potential problem with this way of deleting 
>     // event handlers, basically you can not be sure that you delete 
>     // the event handler that was create by this wxScrollHelper.
> 
>     // Remove the old event handler from the previous target scroll window.
>     if (m_targetWindow && m_targetWindow != m_win)
>         m_targetWindow->PopEventHandler(TRUE /* Delete old event handler*/);
> 
361a373,377
> 
>     // Install a new event handler, which will intercept the events we're
>     // interested in from the target scroll window.
>     if (m_targetWindow != m_win)
>         m_targetWindow->PushEventHandler(new wxScrollHelperEvtHandler(this));
983c999
<         newEvent.SetPosition(m_xScrollPosition - lines);
---
>         newEvent.SetPosition(0);


 	  	 
